lib/osf-components/addon/components/new-feature-popover/component.ts

```diff
@@ -18,6 +18,16 @@ export default class NewFeaturePopover extends Component<Args> {
         return !this.cookies.exists(this.args.featureCookie);
     }
 
+    @action
+    setClickOutsideHandler(element: HTMLElement, [closePopover]:[any]) {
+        const bodyElement = document.querySelector('body');
+        bodyElement?.addEventListener('click', event => {
```

Comment:
The [ember tutorial](https://guides.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_out-of-component-modifications) suggests that, if we go this path, we need to remove the event listener when we close the popover, presumably to keep from sending this click event for the rest of the lifecycle of the app. The same tutorial suggests using ember-modifier, which we have installed, to do this instead in a reusable way. Probably we should do one or the other.